src/components/Navbar/SearchDrawer/SearchDrawer.tsx

```diff
@@ -32,11 +41,13 @@ const VoiceSearchBodyContainer = dynamic(
   },
 );
 
+const PAGE_SIZE = 10;
 const DEBOUNCING_PERIOD_MS = 1000;
 
 const SearchDrawer: React.FC = () => {
   const selectedTranslations = useSelector(selectSelectedTranslations, areArraysEqual);
   const [focusInput, searchInputRef]: [() => void, RefObject<HTMLInputElement>] = useFocus();
+  const [currentPage, setCurrentPage] = useState<number>(1);
```

Comment:
There is not pagination in the side drawer. This logic is for the search page only

Reply:
Oh! some logic was copy-pasted and should have been removed, but I forgot, I will pay more attention next time insha'Allah.